tests/integration/components/au-radio-group-test.js

```diff
@@ -0,0 +1,87 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'dummy/tests/helpers';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | au-radio-group', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it can be given a name', async function (assert) {
+    await render(hbs`
+      <AuRadioGroup @name="foo" as |Group|>
+        <Group.Radio data-test-foo>Foo</Group.Radio>
+      </AuRadioGroup>
+    `);
+    assert.dom('[data-test-foo]').hasAttribute('name', 'foo');
+  });
+
+  test('if it is given no name, it will automatically generate a unique id', async function (assert) {
+    await render(hbs`
+      <AuRadioGroup as |Group|>
+        <Group.Radio data-test-foo>Foo</Group.Radio>
+      </AuRadioGroup>
+    `);
+    assert.dom('[data-test-foo]').hasAttribute('name', /^ember\d+/); // E.g. 'ember308'
+  });
+
+  test('it can be given a value', async function (assert) {
+    this.groupValue = 'bar';
+    await render(hbs`
+      <AuRadioGroup @value={{this.groupValue}} as |Group|>
```

Comment:
I wonder if we should rename `@value` to `@selected` similar to ember-power-select? It looks a bit strange since the radios also use the `@value` argument.

Reply:
I think I took a peek around inside some other frameworks and it seemed like this was largely the default name.

Also, if you need get the selected value of a list in vanilla JS, you'll need something like this: `document.querySelector('input[name="foo"]:checked').value`.

But I'm still a bit on the fence regarding that. If you think it will bring some extra clarity, sure, why not.

Reply:
Maybe I'll ask around within the Kaleidos dev team on Wednesday (Tuesday is my 'day off' every week btw).

Reply:
> Maybe I'll ask around within the Kaleidos dev team on Wednesday (Tuesday is my 'day off' every week btw).

`@selected` it is 👍 

Changed it for both checkbox- and radio-group components.